app/models.py

```diff
@@ -2965,22 +2965,22 @@ def set_approval(self, approved):
                 self.data.pop('representative', None)
                 self.data.pop('phone', None)
                 self.data.pop('email', None)
-
-                if self.data.get('recruiter') == 'no':
-                    self.data['labourHire'] = {}
-                else:
-                    to_remove = []
-                    for state, state_value in self.data.get('labourHire', {}).iteritems():
-                        if not (
-                            state_value.get('expiry') and
-                            state_value.get('licenceNumber')
-                        ):
-                            to_remove.append(state)
-                    for r in to_remove:
-                        self.data.get('labourHire', {}).pop(r)
-
             else:
                 supplier = Supplier()
+
+            if self.data.get('recruiter') == 'no':
```

Comment:
I think we'll have to remove this condition as well, because non recruiters are able to provide licencing info since they are allowed to respond to specialist briefs (this was a late change in the feature, and this code wasn't updated to reflect that).

Reply:
I've removed this condition which has simplified the tests too.